data/start.txt

```diff
@@ -8,7 +8,10 @@
 # WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
 # PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
-start
+start "default"
+	name "Endless Sky"
+	description "The default scenario for Endless Sky"
```

Comment:
```suggestion
	description `You grew up on New Boston, an uninteresting world in the Dirt Belt. You've dreamed of owning a starship ever since your first visit to the spaceport at the age of fifteen. After many long years of working at a textile mill, you've finally saved enough credits to apply for a pilot's license from the Republic.`
	description `    After receiving the license and with freedom from this planet at your fingertips, you travel to the bank to take out a loan on your first ship...`
```